hack/helm/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+
+# Script to test helm charts
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+check() {
+    local TEST_DIR="$1"
+    cd "${TEST_DIR}"
+
+    docker run -ti --rm -v "$(pwd)":/apps quintush/helm-unittest:latest -3 -f 'templates/tests/*.yaml' .
+    cd -
+}
+
+# The following odd style is to resolve SC2045
+shopt -s dotglob; for i in "${SCRIPT_DIR}"/../../deploy/*; do
```

Comment:
I am not sure I follow here SC2045 is "Iterating over ls output is fragile. Use globs." you are not iterating over ls output? Also dotglob does include filenames starting with . in the glob expansion, is that really what we want here?

Reply:
Previously implementation was:
```
for i in $(ls "${SCRIPT_DIR}"/../../deploy/); do
```
hence me moving to the above style to resolve `SC2045`.

I've adjusted the glob to include alphanumeric, and removed the comment.  Hopefully it makes sense as is.